File: API_Server/Taxi.Domain/Models/Drivers/Schedule/ProvinceList.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;

namespace Taxi.Domain.Models.Drivers.Schedule
{
    public static class ProvinceList
    {
        public readonly static Hashtable Province = new Hashtable()
        {
            {"Thanh Hoá", 1 },
            {"Nghệ An", 2 },
            {"Hà Tĩnh", 3 },
            {"Quảng Bình",4 },
            {"Quảng Trị",5 },
            {"Thừa Thiên Huế", 6 },
            {"Đà Nẵng", 7 },
            {"Quảng Nam", 8 },
            {"Quãng Ngãi", 9 },
            {"Bình Định", 10 },
            {"Phú Yên", 11 }
        };
    }
}

File: Client/src/app/shared/models/provinceList.ts
import { Injectable } from '@angular/core';

@Injectable({
  providedIn: 'root'
})
export class Province {
  public provinceList: Object[] = [
    { provinceKey: 1, provinceName: 'Thanh Hoá' },
    { provinceKey: 2, provinceName: 'Nghệ An' },
    { provinceKey: 3, provinceName: 'Hà Tĩnh' },
    { provinceKey: 4, provinceName: 'Quảng Bình' },
    { provinceKey: 5, provinceName: 'Quảng Trị' },
    { provinceKey: 6, provinceName: 'Thừa Thiên Huế' },
    { provinceKey: 7, provinceName: 'Đà Nẵng' },
    { provinceKey: 8, provinceName: 'Quảng Nam' },
    { provinceKey: 9, provinceName: 'Quãng Ngãi' },
    { provinceKey: 10, provinceName: 'Bình Định' },
    { provinceKey: 11, provinceName: 'Phú Yên' },
  ];
  getProvinceList(): Object[]{
    return this.provinceList;
  }
}
